### ISSUE TITLE
Error reporting for files explicitely excluded in config

### ISSUE THREAD

**Issue by designermonkey** [wongjn/vscode-php-sniffer#78]
**Describe the bug**
Even though certain files (for example `vendor/`) are explicitely excluded in the workspace level config, when these files are opened in the editor, they cause the following rror:

```
{ "vscodeOptions": { "tabWidth": 4 }, "result": 
ERROR: No files were checked.
All specified files were excluded or did not match filtering rules.

```

**Expected behavior**
No error should be displayed as those files are supposed to be ignored.

**Environment**
 - OS: macOS Tahoe
 - VSCode version: VSCodium 1.107.18627
 - PHP CLI version: PHP 8.5.2 (cli) (built: Jan 13 2026 21:40:53) (NTS)
 - squizlabs/PHP_CodeSniffer version: PHP_CodeSniffer version 4.0.1 (stable) by Squiz and PHPCSStandards

**Extension settings**
All defaults
